static/app/views/profiling/content.tsx

```diff
@@ -93,7 +93,12 @@ function ProfilingContent({location, selection}: ProfilingContentProps) {
                   </Alert>
                 )}
                 <ProfilingScatterChart
-                  {...dateSelection}
+                  datetime={{
```

Comment:
Small nit: this object is creating a new reference each time. Do we need to cast to null here, or can we just pass datetime={dateSelection} and handle it in the ProfilingScatterChart? This way we have the consistent type signature + no new reference.

Reply:
I opted to use the `datetime` directly from the page filters so that the types are aligned